src/insert/spacefinder/article-body-adverts.ts

```diff
@@ -290,19 +290,35 @@ const addDesktopInline2PlusAds = (): Promise<boolean> => {
 const addMobileInlineAds = (): Promise<boolean> => {
 	const rules: SpacefinderRules = {
 		bodySelector: articleBodySelector,
-		slotSelector: ' > p',
+		candidateSelector: [
```

Comment:
This needs to be an array for multiple elements due to how the body and candidate selector is combined within spacefinder, if they were simply comma separated it would end up for example like `.article-commercial-selector > p, h2, hr` which would select h2 and hr elements that aren't direct children of the article body.